lab-davide/gulpfile.js

```diff
@@ -0,0 +1,61 @@
+var gulp = require('gulp');
+var mocha = require('gulp-mocha');
+var gulp =  require('gulp')
+
+gulp.task('default', function() {
```

Comment:
Instead of having your default gulp task run your tests I suggest creating a separate task for running your tests so you can use your default task for running your test task as well as others (like eslint). :)